fiftyone/operators/_types/types.py

```
@@ -2533,10 +2533,24 @@ class TextFieldView(View):
     .. note::
 
         Must be used with :class:`String` or :class:`Number` properties.
+
+    Args:
+        multiline (False): whether to render a multiline text area
+        rows (None): optional number of visible text rows when ``multiline``
+            is enabled
     """
 
-    def __init__(self, **kwargs):
+    def __init__(self, multiline=False, rows=None, **kwargs):
```

Ruff (0.15.7)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)